docs/dev/types_serialization.md

```diff
@@ -153,7 +153,7 @@ val input = env.fromElements(
     new WordWithCount("hello", 1),
     new WordWithCount("world", 2)) // Case Class Data Set
 
-input.keyBy("word")// key by field expression "word"
+input.keyBy(_.word) // .keyBy("word") is @Deprecated
```

Comment:
In my opinion, there is no point in mentioning something that is deprecated. Let's just drop the comment.